src/Todo.tsx

```diff
@@ -0,0 +1,150 @@
+import React, { useState } from "react";
+import { ListGroup, Form, Modal } from "react-bootstrap";
+import "bootstrap/dist/css/bootstrap.min.css";
+import DateTimePicker from "react-datetime-picker";
+import "react-date-picker/dist/DatePicker.css";
+import "react-calendar/dist/Calendar.css";
+import "react-clock/dist/Clock.css";
+import "react-datetime-picker/dist/DateTimePicker.css";
+import { IoAddCircleOutline } from "react-icons/io5";
+import { CiAlarmOn } from "react-icons/ci";
+import "./App.css";
+
+const Todo: React.FC = () => {
+  const [items, setItems] = useState<TodoItem[]>([]);
+  const [error, setError] = useState<boolean>(false);
+  const [value, onChange] = useState<Value>(new Date());
+  const [show, setShow] = useState<boolean>(false);
+  const [inputValue, setInputValue] = useState<string>("");
+  const [signal, setSignal] = useState<boolean>(false);
+  const [checkInput, setCheckInput] = useState<boolean>(true);
+  type TodoItem = {
+    id: number;
+    text: string;
+    dateTime: Date;
+    completed: boolean;
+    color: string;
+  };
+  type ValuePiece = Date | null;
+  type Value = ValuePiece | [ValuePiece, ValuePiece];
+
+  const handleShow = () => setShow(true);
+  const handleClose = () => {
+    setShow(false);
+  };
```

Comment:
improve code 
```
 const handleShow = () => setShow(true);
  const handleClose = () => {
    setShow(false);
  };```
  i do not thing you need these functions to set show @Rishab-innovative